examples/cifar10/readme.md

```diff
@@ -71,27 +77,32 @@ using namespace tiny_dnn;
 using namespace tiny_dnn::activation;
 
 template <typename N>
-void construct_net(N &nn, core::backend_t backend_type) {
-  typedef convolutional_layer<activation::identity> conv;
-  typedef max_pooling_layer<relu> pool;
-
-  const serial_size_t n_fmaps = 32;  ///< number of feature maps for upper layer
-  const serial_size_t n_fmaps2 =
-    64;  ///< number of feature maps for lower layer
-  const serial_size_t n_fc =
-    64;  ///< number of hidden units in fully-connected layer
-
-  nn << conv(32, 32, 5, 3, n_fmaps, padding::same, true, 1, 1, backend_type)
-     << pool(32, 32, n_fmaps, 2, backend_type)
-     << conv(16, 16, 5, n_fmaps, n_fmaps, padding::same, true, 1, 1,
-             backend_type)
-     << pool(16, 16, n_fmaps, 2, backend_type)
-     << conv(8, 8, 5, n_fmaps, n_fmaps2, padding::same, true, 1, 1,
-             backend_type)
-     << pool(8, 8, n_fmaps2, 2, backend_type)
-     << fully_connected_layer<activation::identity>(4 * 4 * n_fmaps2, n_fc,
-                                                    true, backend_type)
-     << fully_connected_layer<softmax>(n_fc, 10, true, backend_type);
+void construct_net(N& nn, core::backend_t backend_type) {
+    using conv = convolutional_layer;
+    using pool = max_pooling_layer;
+    using fc = fully_connected_layer;
+    using relu = relu_layer;
+    using softmax = softmax_layer;
+
+  const serial_size_t n_fmaps  = 32;  // number of feature maps for upper layer
```

Comment:
Yes, I think we should clearly do a complete cleanup of project and decide where to use `size_t` and where to use `serial_size_t`.